.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/python/black
+    rev: stable
+    hooks:
+    - id: black
+      language_version: python3.6
```

Comment:
what happens when you only have python 3.7 installed and try to run this?

Reply:
you may also want to look into python/black#675 and add `require_serial: true`

Reply:
seems reasonable, i will be adding for now. we can update later if we observe non-serial is indeed better